src/game.js

```diff
@@ -36,6 +36,15 @@
     this.ball.draw(this.ball.x, this.ball.y);
   };
 
+  Game.prototype.collisionDetection = function () {
+    if (this.ball.y + this.ball.dy - this.ball.radius < 0) {
```

Comment:
can the contents of the if clause be moved into a private method that returns true or false? 
e.g. `if ( ballWillBounce() )`